nautobot/ipam/tests/test_views.py

```diff
@@ -266,10 +225,10 @@ def test_empty_queryset(self):
         but the same behavior was observed in other filters, such as IPv4/IPv6.
         """
         prefixes = self._get_queryset().all()
-        self.assertEqual(prefixes.count(), 3)
+        self.assertNotEqual(prefixes.count(), 0)
 
         url = self._get_url("list")
-        response = self.client.get(f"{url}?status=deprecated")
+        response = self.client.get(f"{url}?status=nonexistentstatus")
```

Comment:
This will trip over STRICT_FILTERING won't it? Maybe better to create a one-off Status that is valid for Prefix but doesn't have any assigned to it.

Reply:
Confirmed that it does break on "Invalid filters were specified"